js/os.ts

```diff
@@ -23,6 +23,41 @@ interface CodeInfo {
   sourceMap: string | undefined;
 }
 
+interface IsTTY {
+  stdin: boolean;
+  stdout: boolean;
+  stderr: boolean;
+}
+
+let _isTTY: IsTTY; // cache
+
+/** Check if running in terminal.
+ *
+ * isTTY().stdin; // stdin
+ * isTTY().stdout; // stdout
+ * isTTY().stderr; // stderr
+ */
+export function isTTY(): IsTTY {
```

Comment:
Just inline the interface here `{ stdin: boolean, stdout: boolean, stderr: boolean }`

Reply:
not resolved...

Reply:
```ts
export function isTTY(): { stdin: boolean, stdout: boolean, stderr: boolean } {
  // ...
}
```